scss/inc/_grid.scss

```diff
@@ -17,9 +17,15 @@
     overflow-x: hidden; 
 }
 
-.grid-row, .fixed-grid-row {
+.grid-row, .fixed-grid-row, .colrow {
     @extend .clearfix;
     width: widthPerc(852, 840);
+    &:before {

```

Comment:
https://github.com/oat-sa/tao-core-ui-fe/blob/ab598265609cd90ee148c7bf2e2a0732e13e2ad3/scss/inc/_variables.scss#L13-L21
before and after already included in extend %clearfix